public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -59,12 +61,21 @@ export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, dat
                   />
                 ) : null}
                 <h2 className={styles.titleText}>{title}</h2>
-                <Icon name="angle-down" className="panel-menu-toggle" />
-                <PanelHeaderMenuWrapper panel={panel} dashboard={dashboard} show={panelMenuOpen} onClose={closeMenu} />
-                {data.request && data.request.timeInfo && (
-                  <span className="panel-time-info">
-                    <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
-                  </span>
+                {!isPublicDashboardView() && (
```

Comment:
this makes it impossible to view a panel in full screen view

Reply:
It's better to remove all but view panel actions from the menu

Reply:
getPanelMenu  get's the dashboard, it can check if it's a public dashboard in the dashboard meta object (Where other permission flags are) , we should not use the URL to check if it's a public dashboard, should be a flag on dashboard.meta   (and a flag on the user object in contextSrv.user)

Reply:
We decided to remove all menu options for now to keep things simple. There's still a lot of UX items that are undecided (like this one).

I'm not sure if we can use the dashboard.meta flag for this. If you're a non-public user looking at dashboard (maybe you're making dashboard changes or something), you should still be able to work with the dashboard as usual, even though its marked as 'public'.

Reply:
This looks like a messier option in my view, we already have conditional logic for what menu items to show. sure hide it there are no menu items.